ingestion/src/metadata/ingestion/source/dashboard/looker/bulk_parser.py

```diff
@@ -0,0 +1,148 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+.lkml files parser
+"""
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import lkml
+from metadata.ingestion.source.dashboard.looker.models import (
+    Includes,
+    LkmlFile,
+    LookMlView,
+    ViewName,
+)
+from metadata.readers.file.base import ReadException
+from metadata.utils.logger import ingestion_logger
+
+
+from ingestion.src.metadata.readers.file.api_reader import ApiReader
+from ingestion.src.metadata.readers.file.local import LocalReader
```

Comment:
these imports are going to fail once the package is installed. Instead of `ingestion.src.metadata` it should directly be `metadata....`

Reply:
i've seen them in a few places

Reply:
Import: Updated